### PR TITLE
Adds more information about ingest attachment properties extraction

### PR DESCRIPTION
**Pull request by dadoonet**
This is coming from this thread on discuss: https://discuss.elastic.co/t/ingest-attachment-plugin-exception/69167/10

### CODE REVIEW

File: docs/plugins/ingest-attachment.asciidoc [hunk 1]
```diff
@@ -102,6 +102,25 @@ Returns this:
 --------------------------------------------------
 // TESTRESPONSE
 
+
+To specify only some fields to be extracted:
+
+[source,js]
+--------------------------------------------------
+PUT _ingest/pipeline/attachment
```
Comment (spinscale) [inline docs/plugins/ingest-attachment.asciidoc]: you might want to add `// CONSOLE` here at the end as well?

**Comment by spinscale** [comment]
building the docs fails with this PR

**Comment by dadoonet** [comment]
@spinscale `build_docs.pl --doc index.asciidoc --open` works. You mean gradle build right?

**Comment by dadoonet** [comment]
I see `gradle check` fails indeed.
Fixing...

**Comment by dadoonet** [comment]
@spinscale `gradle check` is now happy again... Thanks!

File: docs/plugins/ingest-attachment.asciidoc [hunk 2]
```diff
@@ -52,7 +52,7 @@ The node must be stopped before removing the plugin.
 | `field`                | yes       | -                | The field to get the base64 encoded field from
 | `target_field`         | no        | attachment       | The field that will hold the attachment information
 | `indexed_chars`        | no        | 100000           | The number of chars being used for extraction to prevent huge fields. Use `-1` for no limit.
-| `properties`           | no        | all              | Properties to select to be stored. Can be `content`, `title`, `name`, `author`, `keywords`, `date`, `content_type`, `content_length`, `language`
+| `properties`           | no        | all properties   | Properties to select to be stored. Can be `content`, `title`, `name`, `author`, `keywords`, `date`, `content_type`, `content_length`, `language`
```
Comment (spinscale) [inline docs/plugins/ingest-attachment.asciidoc]: maybe mention here, that this is supposed to be an array. I think that was one of the missing pieces in that forum post, that there was no clear way how to specify this?

**Comment by spinscale** [comment]
left one last comment, LGTM otherwise

**Review by spinscale — APPROVED** [review]

File: docs/plugins/ingest-attachment.asciidoc (hunk 2 above)
Comment (dadoonet) [inline docs/plugins/ingest-attachment.asciidoc]: I agree. I wondered first if we should add somehow a new column with the expected format, or add it within the comment ro as an example which I did at the end.
I cand replace `Properties to select` with `Array of properties to select`?

**Pull request merged**